gutterball/src/test/java/org/candlepin/gutterball/junit/GutterballLiquibaseResource.java

```diff
@@ -158,5 +158,8 @@ public void truncateLiquibaseSchema() {
     public void createLiquibaseSchema() {
         exec("CREATE SCHEMA LIQUIBASE");
         database.setLiquibaseSchemaName("LIQUIBASE");
+
+        // HSQLDB does not support mediumtext, so we fake it here.
+        exec("CREATE TYPE MEDIUMTEXT AS VARCHAR(100000)");
```

Comment:
Interesting tidbit I learned while looking up if HSQLDB has a limit on varchar: You can use K, M or G in the length specifier, like VARCHAR(10k)
http://hsqldb.org/doc/guide/sqlgeneral-chapt.html#sgc_char_types